src/sidebar/components/Thread.js

```diff
@@ -90,24 +91,29 @@ function Thread({ thread, threadsService }) {
     >
       {showThreadToggle && (
         <div className="Thread__collapse">
-          <Button
-            className="Thread__collapse-button"
-            icon={toggleIcon}
-            title={toggleTitle}
-            onClick={onToggleReplies}
-          />
+          <div className="Thread__collapse-button-container">
+            <IconButton
+              className="NonResponsiveIconButton"
+              expanded={!thread.collapsed}
+              icon={toggleIcon}
+              title={toggleTitle}
+              onClick={onToggleReplies}
+              size="medium"
+              variant="light"
```

Comment:
Note use of `light` variant here! I also thought I'd use `size="small"` here, but `medium` had a more appropriate target size.